chapter3.adoc

```diff
@@ -3,8 +3,12 @@
 == `Smsdid`: Supervisor Domain Identifier and Protection Register
 
 `Smsdid` defines an interface to program the active supervisor domain under
-which a hart is operating. The interface consists of M-mode CSRs `msdcfg` and
-`mmpt`. The SDID programmed via this interface is a local identifier for the
+which a hart is operating and the configuration of the active supervisor domain.
```

Comment:
> `Smsdid` defines an interface to program the active supervisor domain under

defines an interface to program a supervisor domain ID (SDID) and configurations
for supervisor domain related extensions?